source/3d-secure-2.html.erb

```diff
@@ -83,7 +87,7 @@ title: GOV.UK Pay 3D Secure 2
         <li>corporate payments (unless the corporate card is in the name of an individual)</li>
       </ul>
 
-      <p class="govuk-body">Which exemptions are applied will depend on your Payment Service Provider (PSP).</p>
+      <p class="govuk-body">Which exemptions are applied will depend on your Payment Service Provider (PSP). See ‘How to set up 3D Secure 2’ for more details.</p>
```

Comment:
Should ‘How to set up 3D Secure 2’ be linked?

Reply:
Could add an anchor link?

Reply:
If there is one, yes

Reply:
OK, added with a new commit!